linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -1256,6 +1265,8 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
 }
 
 private object ClassEmitter {
+  private final val ThrowableClass = "jl_Throwable"
```

Comment:
I'm a bit sad that we re-introduce another special type which is not a hijacked class. But since `Throwable` can have subclasses (which is not the case for any hijacked class), this is probably necessary.

Reply:
It is way less special than hijacked classes, though, because `Throwable` does not affect the typing rules of the IR.

The closest special type in the IR is `java.lang.Class`. With a cursory glance, it doesn't seem like it *should* be special, but it is the result type of the `ClassOf` and `GetClass` nodes, so it does appear in the typing rules of the IR.

Reply:
> It is way less special than hijacked classes, though, because Throwable does not affect the typing rules of the IR.

Good point. Fair enough, so we can really keep this as an emitter implementation detail. Maybe we could guard the feature by checking that `Throwable` extends `Object` directly and not something else? (IIUC it would be totally valid IR to have a `jl_Throwable` which is something totally different than the java one).